### PR TITLE
Remove dead stores

### PR DESCRIPTION
**Pull request by SunBlack**
Remove dead stores reported by `run-clang-tidy -header-filter='.*' -checks='-*,clang-analyzer-deadcode.DeadStores'`

Addition: In svm.cpp removed `endptr`, even it was not detected by clang.

### CODE REVIEW

**Comment by SunBlack** [comment]
This PR is ready to merge (MacOS build just failed without any reason).

File: tools/tiff2pcd.cpp [hunk 1]
```diff
@@ -394,10 +392,6 @@ int main(int argc, char ** argv)
           // Continue with the next depth entry
           continue;
         }
-        if(found == 0)
```
Comment (taketwo) [inline tools/tiff2pcd.cpp]: I think this is rather a bug. This check `found == 0` and following console output should happen outside of the loop.

Comment (SunBlack) [reply tools/tiff2pcd.cpp]: Adjusted

**Comment by taketwo** [comment]
This has only trivial changes, so I'll merge this before it accumulates even more conflicts.

**Pull request merged**